ga4gh/serverconfig.py

```diff
@@ -44,7 +46,9 @@ class LocalOidConfig(DevelopmentConfig):
     """
     Configuration used for developing against a local OIDC server
     """
-    OIDC_PROVIDER = "https://localhost:8443"
+    OIDC_PROVIDER = "https://{}:8443".format(socket.gethostname())
+    PERMISSIONS = {'diana@example.org': ['d1'],
```

Comment:
I think `PERMISSIONS` must be empty by default. It's fine having TestConfig and (say) DevelopmentConfig having some hard-coded permissions to make our lives easier, but in general we need to think of users actually filling out a config file.